app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,55 @@
+class ReviewsController < ApplicationController
+  before_action :authenticate_user!, except: [:index, :show]
+
+  def show
+    @review = Review.find(params[:id])
+    @pizzeria = Pizzeria.find(@review.pizzeria_id)
```

Comment:
You can simplify this by using the association:

```ruby
@pizzeria = @review.pizzeria
```

I wouldn't even both putting it in an instance variable though. You can just reference `@review.pizzeria` in your view.